src/patternfly/components/SimpleList/simple-list.scss

```
@@ -14,7 +14,11 @@
   --#{$simple-list}__item-link--hover--Color: var(--pf-t--global--text--color--subtle);
   --#{$simple-list}__item-link--hover--BackgroundColor: var(--pf-t--global--background--color--action--plain--hover);
   --#{$simple-list}__item-link--BorderRadius: var(--pf-t--global--border--radius--tiny);
-
+  --#{$simple-list}__item-link--BorderWidth: 0;
+  --#{$simple-list}__item-link--BorderColor: var(--pf-t--global--border--color--high-contrast);
+  --#{$simple-list}__item-link--hover--BorderWidth: var(--pf-t--global--border--width--regular);
+  --#{$simple-list}__item-link--current--BorderWidth: var(--pf-t--global--border--width--strong);
```

Just a nit, if the var is for a modifier class `.pf-m-[something]`, the var scheme is `--m-[something`

```suggestion
  --#{$simple-list}__item-link--m-current--BorderWidth: var(--pf-t--global--high-contrast--border--width--strong);
```